packages/app/src/runs/RunsLayout.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div
+    v-if="isUsingGit"
+    class="flex flex-col gap-[24px]"
+  >
+    <ul
+      data-cy="runsLayout-git"
+      class="flex flex-col gap-[16px] relative before:content-[''] before:absolute before:top-[20px] before:bottom-[10px] before:w-[2px] before:border-2 before:border-dashed before:border-l-0 before:border-y-0 before:border-r-gray-100 before:left-[7px]"
+    >
+      <li
+        v-for="sha of Object.keys(groupByCommit)"
+        :key="sha"
+        :data-cy="`commit-${sha}`"
+        :class="`${!groupByCommit[sha].runs ? 'mb-[-24px]' : ''}`"
+      >
+        <div class="flex items-center my-[10px] [&>*:last-child]:mr-[8px]">
```

Comment:
Minor, but this class `my-[10px]` causes there to be too much whitespace at the top of the run page layout in my opinion.

![Screenshot 2023-09-12 at 10 51 36 AM](https://github.com/cypress-io/cypress/assets/14275198/b1b75566-8375-4e21-ad39-8ee2360b9c10)

Could add something like this to get rid of the top margin only for the first commit in the list
```
v-for="(sha, index) of Object.keys(groupByCommit)"
```

```
:class="{ 'mt-0': index === 0 }"
```

Reply:
Updated.